.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -78,10 +78,10 @@ schedules:
 variables:
   buildConfiguration: Release
   dotnetCoreSdkLatestVersion: 6.0.100
-  relativeTracerHome: /tracer/src/bin/windows-tracer-home
+  relativeTracerHome: /shared/bin/monitoring-home/Tracer
```

Comment:
nit: I know it would be super-annoying to change, but any reason `Tracer` needs to be uppercase (here and everywhere else)? It's triggering my OCD 😅

Reply:
The reason it's like that is because that's the path the Windows MSI is using (and consequently the one currently used in `loader.conf`). I don't mind changing it at all, but if we're changing it I see only two options here:

1. Change it to lower-case (i.e.``monitoring-home/tracer` and `monitoring-home/debugger`), which would make it look consistent in on Linux and in our git repo, but make it look inconsistent on Windows (e.g. `C:\Program Files\Datadog .NET Tracer\continuous-profiler`)
2. Use upper-case on Windows and lower-case on Linux, which would mean we'd have two different `loader.conf` files, which I was hoping we can avoid. 

@zacharycmontoya do you happen to have any feedback on this?

Reply:
All lowercase is fine by me. I don't recall strong motivations for making it uppercase originally. Perhaps gregp previously advocated for it but I support changing to lowercase

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2358/commits/dd2a1ee47b3c8fab93dc2f8545a8733ffc234345